Automattic/wp-calypso
# Set default share image for coming soon pages

#### Proposed Changes

This PR changes the default share image for sites whose Privacy is set to Coming soon.

If no site icon is set, the share image will be this image:

![image](https://user-images.githubusercontent.com/528287/204283613-3de482f9-cb74-4904-9a02-4a698d01fee0.png)


#### Testing Instructions

Follow the ETK plugin testing instructions: PCYsg-ly5-p2

After applying this PR you can:

- Set your site's Privacy to Coming soon (Settings > General)
- Remove site icon if any
- Open your site's homepage
- View source
- Ensure these two (separate) lines are there:

```<meta property="og:image" content="https://s1.wp.com/home.logged-out/images/wpcom-og-image.jpg" />```

```<meta name="twitter:image" content="https://s1.wp.com/home.logged-out/images/wpcom-og-image.jpg" />```

It's not possible to validate using the Facebook Sharing Debugger, since Facebook's DNS doesn't point to your sandbox.

#### Pre-merge Checklist


Related to #62773
